### PR TITLE
Download page adjustments

### PR DESCRIPTION
**Pull request by samdark**
- Less text
- No fxp
- Mentioned composer for 1.1

### CODE REVIEW

File: views/site/download.php [hunk 1]
```diff
@@ -193,12 +173,6 @@
 					<h2 id="offline-documentation"><span>Offline Documentation</span></h2>
 				</div>
 
-				<p class="small">
-					The Definitive Guide to Yii 2.0 and the API Documentation are available for offline browsing.
-					Below you will find the download packages including the HTML files of Guide and API Documentation
-					as well as links to the PDF versions of the Guide in different languages.
-				</p>
```
[the remark this answers is not in the document]
Comment (samdark) [reply views/site/download.php]: Isn't it obvious by looking at the links?

File: views/site/download.php [hunk 2]
```diff
@@ -26,44 +26,36 @@
...
+                    If you do not have Composer installed yet,
+                    you may install it following the instructions
+                    on the <a href="https://getcomposer.org/download/">Composer website</a>.
+                </p>
+            </div>
+
 			<div class="col-md-12">
-				<p>
-					Yii is an open source project released under the terms of the <?= Html::a('BSD License', ['site/license']) ?>.
-					This means that you can use Yii for free to develop either open-source or proprietary Web applications.
```
[the remark this answers is not in the document]
Comment (samdark) [reply views/site/download.php]: Added above.

File: views/site/download.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (cebe) [reply views/site/download.php]: PDF is good, but I would not know what to expect from a `.tar.bz2` file...

File: views/site/download.php [hunk 3]
```diff
@@ -26,44 +26,36 @@
...
+            <li><?= Html::a('Upgrade instructions', "https://github.com/yiisoft/yii2/blob/$latest/framework/UPGRADE.md") ?></li>
+            <li><a href="https://github.com/yiisoft/yii2/releases">List of all Yii 2.x releases</a></li>
+        </ul>
 
-<!--            <div class="row text-center ptrem2">
-					<div class="col-md-12">
-					<p class="small"><a href="#changelog">Complete change log</a><span class="separator">|</span><a href="#upgrade">Upgrade instructions</a></p>
-					</div>
-				</div>-->
+				<h1 id="install-composer">Install via Composer (recommended)</h1>
```
[the remark this answers is not in the document]
Comment (samdark) [reply views/site/download.php]: This one was kept as is. We can adjust markdown later as styles are bound to it.

[the remark this answers is not in the document]
Comment (cebe) [reply views/site/download.php]: ok

File: views/site/download.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (samdark) [reply views/site/download.php]: Fixed in the links themselves. They now say `HTML in .tar.bz2`

[the remark this answers is not in the document]
Comment (cebe) [reply views/site/download.php]: well okay, this does not say anything about guide and API but okay. no strong opinion on this one.

**Comment by cebe** [comment]
merged, thank you!

**Pull request closed without merging**